src/segmented_cache.erl

```diff
@@ -70,7 +70,7 @@
 -spec is_member(name(), term()) -> boolean().
 is_member(Name, Key) when is_atom(Name) ->
     Span = fun() ->
-                   Value = iterate_fun_in_tables(Name, Key, fun ?MODULE:is_member_fun/2),
+                   Value = ?HELPERS:iterate_fun_in_tables(Name, Key, fun ?HELPERS:is_member_fun/2),
```

Comment:
this can be wrapped in a separate function `?HELPER:is_member()`,  and if the same is done for `get_entry/2`, `put_entry/2`, `delete_entry/2` and `delete_pattern/2` then we don't need to export `iterate_fun_in_tables/3` function as well as related callbacks from the ?HELPER module.

Reply:
also, callbacks can be moved into a separate module as well.